index.js

```diff
@@ -113,6 +124,12 @@ function log(instanceLog) {
 
     // Assign the _label
     data._label = level.toUpperCase();
+
+    // Include heap info if configured.
+    if (process.env.CLAY_LOG_HEAP === '1') {
```

Comment:
do you need to do this strict check for value or will `if (process.env.CLAY_LOG_HEAP)` be sufficient?

Reply:
So the problem with just checking `process.env.CLAY_LOG_HEAP` is that *any* truthy value will enable this feature.

Since environment variables are strings, that means `"0"` would also enable the feature.

In order to allow people to explicitly *disable* this feature we need to be a little stricter about what values we accept as `true`.